### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/advanced/sub-applications.md` and `docs/pt/docs/advanced/behind-a-proxy.md`

### PR DESCRIPTION
**Pull request by marcelomarkus**
- **Add translation:**
docs/pt/docs/advanced/sub-applications.md
docs/pt/docs/advanced/behind-a-proxy.md

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 195c2ae1135c20d477b1ea88e43c7ad1ff34d2fe at: https://38176437.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit dd45453428478ca64d48220f70c31473b720d128 at: https://ee1b34d7.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 5b678fd66560aab0f42faf74f8fe3aa63f46adfd at: https://6da2e833.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 71134c47dc9e1d25680cb9f4e90be014d29e96c4 at: https://25bce513.fastapitiangolo.pages.dev

File: docs/pt/docs/advanced/sub-applications.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Se você precisar ter duas aplicações FastAPI independentes, cada uma com seu próprio OpenAPI e suas próprias interfaces de documentação, você pode ter um aplicativo principal e "montar" uma (ou mais) sub-aplicações.
+
+## Montando uma aplicação **FastAPI**
+
+"Montar" significa adicionar uma aplicação completamente "independente" em um caminho específico, que então se encarrega de lidar com tudo sob esse caminho, com as operações de caminho declaradas nessa sub-aplicação.
+
+### Aplicação de nível superior
+
+Primeiro, crie a aplicação principal, de nível superior, **FastAPI**, e suas *operações de caminho*:
```
Comment (ceb10n) [inline docs/pt/docs/advanced/sub-applications.md:11]: Most translations in pt docs are using operações de rota when translating path operations

File: docs/pt/docs/advanced/sub-applications.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+```Python hl_lines="3  6-8"
+{!../../../docs_src/sub_applications/tutorial001.py!}
+```
+
+### Sub-aplicação
+
+Em seguida, crie sua sub-aplicação e suas *operações de caminho*.
```
Comment (ceb10n) [inline docs/pt/docs/advanced/sub-applications.md:19]: Most translations in pt docs are using operações de rota when translating path operations

File: docs/pt/docs/advanced/sub-applications.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+$ uvicorn main:app --reload
+
+<span style="color: green;">INFO</span>:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
+```
+
+</div>
+
+E abra a documentação em <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a>.
+
+Você verá a documentação automática da API para a aplicação principal, incluindo apenas suas próprias _operações de caminho_:
```
Comment (ceb10n) [inline docs/pt/docs/advanced/sub-applications.md:53]: Most translations in pt docs are using operações de rota when translating path operations

File: docs/pt/docs/advanced/sub-applications.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+<img src="/img/tutorial/sub-applications/image01.png">
+
+E então, abra a documentação para a sub-aplicação, em <a href="http://127.0.0.1:8000/subapi/docs" class="external-link" target="_blank">http://127.0.0.1:8000/subapi/docs</a>.
+
+Você verá a documentação automática da API para a sub-aplicação, incluindo apenas suas próprias _operações de caminho_, todas sob o prefixo de sub-caminho correto `/subapi`:
```
Comment (ceb10n) [inline docs/pt/docs/advanced/sub-applications.md:59]: Most translations in pt docs are using operações de rota when translating path operations

**Review by ceb10n — CHANGES_REQUESTED** [review]
Great translation ✨

**Review by lucasbalieiro — COMMENTED** [review]
Tops! Just awaiting the fixes that @ceb10n highlighted.

**Comment by tiangolo** [comment]
📝 Docs preview for commit 59478911240a59a22b23210934210bb70c57b806 at: https://61fe8430.fastapitiangolo.pages.dev

**Review by ceb10n — APPROVED** [review]
🚀

**Comment by tiangolo** [comment]
📝 Docs preview for commit 5c696f067e10fda7c22b4c736d9c46756ce95fe4 at: https://9eefbe8e.fastapitiangolo.pages.dev

**Review by lucasbalieiro — APPROVED** [review]
Let's go

**Comment by alejsdev** [comment]
Yay! thanks for the help! @marcelomarkus  @ceb10n @lucasbalieiro :nerd_face: :v:

**Pull request merged**